policy-controller/k8s/index/src/inbound/tests/authorization_policy.rs

```diff
@@ -264,6 +266,129 @@ fn links_authorization_policy_with_service_account() {
     );
 }
 
+#[test]
+fn authorization_policy_prevents_index_deletion() {
+    let test = TestConfig::default();
+
+    // Create policy resources: Server, HttpRoute, AuthorizationPolicy, and NetworkAuthentication.
+    let server = mk_server(
+        "ns-0",
+        "srv-8080",
+        Port::Number(8080.try_into().unwrap()),
+        None,
+        Some(("app", "app-0")),
+        Some(k8s::policy::server::ProxyProtocol::Http1),
+    );
+    test.index.write().apply(server.clone());
+
+    let authz = ClientAuthorization {
```

Comment:
this is really nitpicky but it was immediately clear to me that this is only used at the end as the expected configuration. Maybe moving it closer to the assert or naming it more explicitly might help. Ultimately once you go through this test top-to-bottom once or twice it starts to add up anyway.